src/bin/tikv-dump.rs

```diff
@@ -125,12 +132,13 @@ fn dump_range(db: DB, from: String, to: Option<String>, limit: Option<u64>) {
     }
 
     let mut cnt = 0;
-    db.scan(&from,
-              &to,
-              &mut |k, v| {
-                  println!("key: {}, value: {}", escape(k), escape(v));
-                  cnt += 1;
-                  Ok(cnt < limit)
-              })
+    db.scan_cf(cf,
+                 &from,
```

Comment:
The style stays after executing `make fmt` again. Maybe caused by some rust-fmt's bug.